server/player/player.go

```diff
@@ -2615,12 +2646,16 @@ func (p *Player) OnGround() bool {
 	return p.onGround.Load()
 }
 
-// EyeHeight returns the eye height of the player: 1.62, or 0.52 if the player is swimming.
+// EyeHeight returns the eye height of the player.
```

Comment:
Could keep the last part of the doc and say "0.52 if the player is crawling or swimming, 1.26 when sneaking or 1.62 when standing."